pkg/3scale/amp/component/backend.go

```diff
@@ -398,6 +437,9 @@ func (backend *Backend) buildBackendWorkerEnv() []v1.EnvVar {
 		)
 	}
 
+	if backend.Options.RedisTLSEnabled {
```

Comment:
we already have these in buildBackendCommonEnv - common in my understanding applies to both, worker and listener - is this really required?

Reply:
Fixed. Thank you